modules/ROOT/pages/ocis_release_notes.adoc

```diff
@@ -23,6 +23,50 @@ You can read more about the Infinite Scale beta phase on https://owncloud.com
 - https://owncloud.com/infinite-scale-beta[Infinite Scale Beta Page]
 - https://owncloud.com/news/infinite-scale-beta/[Infinite Scale Beta is coming]
 
+=== Beta 8
+
+Infinite Scale 2.0.0 Beta 8 includes a huge number of smaller bug fixes and round-offs all around. It introduces a more efficient and scalable way to store share information and improves the performance of ownCloud Web in many areas very noticeable. Furthermore it is now possible to set user quotas in the UI and administrators can now delete orphaned Spaces.
+
+IMPORTANT: When updating from Infinite Scale Beta 7 or lower to Beta 8 it is necessary to run two migration steps from the previous to the new share manager **before** starting the system again regularly.
+
+**Migration for User/Group Shares**
+
+`OCIS_LOG_LEVEL="debug" OCIS_LOG_PRETTY="true" OCIS_LOG_COLOR="true" ocis migrate shares --from cs3 --to jsoncs3`
+
+**Migration for Public Links**
+
+`OCIS_LOG_LEVEL="debug" OCIS_LOG_PRETTY="true" OCIS_LOG_COLOR="true" ocis migrate publicshares --from cs3 --to jsoncs3`
+
+If you have set up your system with Docker, you can just use the same syntax as with https://doc.owncloud.com/ocis/next/deployment/container/container-setup.html#first-time-start[ocis init]:
+
+```
+docker run --rm -it \
+    --mount type=bind,source=$PWD/ocis/ocis-config,target=/etc/ocis \
+    owncloud/ocis migrate shares
```

Comment:
if we can use `ocis migrate shars`, why not use it above? There you stated 2 different share migration commands for user/group and public shares... 🤔

Reply:
I gues you mean `ocis migrate shares`... (or?)

Reply:
Congratulations, you found a typo. 😛 Question itself should be clear, even with the typo.

Reply:
Well, I only provided one example as you just need to replace `shares` with `publicshares`. Thought that transfer would be clear. Should I really add both?

Reply:
Added both now also for the docker commands.